TestProject/SimpleTests.cs.expected

```diff
@@ -0,0 +1,205 @@
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using NUnit.Framework;
+using Rhino.Mocks;
+using TestProject.TestClasses;
+using TestProject.TestInterfaces;
+using Moq;
+using MockRepository = Rhino.Mocks.MockRepository;
+
+namespace TestProject
+{
+  [TestFixture]
+  public class SimpleTest
+  {
+    private Mock<ITestInterface> _mock1;
+    private Mock<ITestInterface> _mock2;
+
+    [SetUp]
+    public void Setup ()
+    {
+    }
+
+    [Test]
+    public void Rewrite_MockInstantiation ()
+    {
+
+      _ = new Mock<ITestInterface>();
+      _ = new Mock<ITestInterface>();
+      _ = new Mock<ITestInterface> (MockBehavior.Strict);
+      _ = new Mock<ITestInterface>() { CallBase = true };
+
+      _ = new Mock<ITestInterface>();
+      _ = new Mock<ITestInterface>();
+      _ = new Mock<ITestInterface> (MockBehavior.Strict);
+      _ = new Mock<ITestInterface> (MockBehavior.Strict);
+      _ = new Mock<ITestInterface>();
+      _ = new Mock<ITestInterface>() { CallBase = true };
+      _ = new Mock<ITestInterface>() { CallBase = true };
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_VerifyAll ()
+    {
+      _mock1 = new Mock<ITestInterface> (MockBehavior.Strict);
+      _mock2 = new Mock<ITestInterface>();
+
+      _mock1.Verify();
+
+      _mock2.Verify();
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_VerifyAllExpectations ()
+    {
+      _mock1.Verify();
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_AssertWasCalled ()
+    {
+      _mock1.Verify (_ => _.Read(), Times.AtLeastOnce());
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_AssertWasNotCalled ()
+    {
+      _mock1.Verify (_ => _.Read(), Times.Never());
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_RepeatAny ()
+    {
+      _mock1.Setup (m => m.Read (null)).Verifiable();
+
+      _mock1.Verify();
+
+      Assert.Pass();
+    }
+
+    [Test]
+    public void Rewrite_RepeatOnce ()
+    {
+      _mock1.Setup (m => m.Read (null)).Verifiable();
+
```

Comment:
Should we add a non-related statement after the Expect (before the verify) to demonstrate the complicated verify sttement is added in the correct place?

Reply:
Since the code wasn't changed and there wasn't an answer, I'd like to clarify the intentions before resolving this.

Reply:
![image](https://user-images.githubusercontent.com/48289242/118251740-3ca6c580-b4a8-11eb-8de2-c16fa9760674.png)

the code was changed, i don't know why the changes are not showing up

Reply:
Okay, this change does show up, it's just not obvious when you don't look at a diff. Anyhow, I was going for a different effect (see code-change comment)